src/bin/hexyl.rs

```diff
@@ -104,24 +104,46 @@ fn run() -> Result<(), AnyhowError> {
                 .takes_value(true)
                 .value_name("WHEN")
                 .possible_values(&["always", "auto", "never"])
+                .default_value_if("plain", None, "never")
                 .default_value("always")
                 .help(
                     "When to use colors. The auto-mode only displays colors if the output \
                      goes to an interactive terminal",
                 ),
         )
+        .arg(
+            Arg::with_name("plain")
+                .short("p")
+                .long("plain")
+                .help("Display output with --no-chars, --no-position, and --border=none."),
+        )
         .arg(
             Arg::with_name("border")
                 .long("border")
                 .takes_value(true)
                 .value_name("STYLE")
                 .possible_values(&["unicode", "ascii", "none"])
+                .default_value_if("plain", None, "none")
                 .default_value("unicode")
                 .help(
                     "Whether to draw a border with Unicode characters, ASCII characters, \
                     or none at all",
                 ),
         )
+        .arg(
+            Arg::with_name("no_chars")
+                .short("C")
+                .long("no-chars")
+                .conflicts_with("plain")
+                .help("Whether to display the character table on the right."),
+        )
+        .arg(
+            Arg::with_name("no_position")
```

Comment:
Same questions here: do we have a similar flag in other hex viewers? If so, how is it called? If there is nothing similar, maybe `--no-file-position`?

Reply:
I'd suggest to avoid using the term file esp. when pipe operations like the one below are somewhat common (in my workflow) to use:
```
$ echo abcdef1234567890 | hexyl
┌────────┬─────────────────────────┬─────────────────────────┬────────┬────────┐
│00000000│ 61 62 63 64 65 66 31 32 ┊ 33 34 35 36 37 38 39 30 │abcdef12┊34567890│
│00000010│ 0a                      ┊                         │_       ┊        │
└────────┴─────────────────────────┴─────────────────────────┴────────┴────────┘
```

`hexdump` seems to use a [formatter](https://stackoverflow.com/questions/2614764/how-to-create-a-hex-dump-of-file-containing-only-the-hex-characters-without-spac) over any toggling, which is fine but much more work:

```
$ echo 'abcdef1234567890' | hexdump -ve '1/1 "%.2x"'
616263646566313233343536373839300a
```

perhaps `--no-position-table` or `--no-byte-hex-table`?

Reply:
Let's go with `--no-position` for now.